lightwood/api/json_ai.py

```diff
@@ -391,19 +392,20 @@ def generate_json_ai(
             ]
         )
         nr_mixers = len(list(outputs.values())[0].mixers)
-        encoder_time_budget_pct = max(
-            3.3 / 5, 1.5 + np.log(nr_trainable_encoders + 1) / 5
-        )
+        encoder_time_budget_pct = (2 + np.log(nr_trainable_encoders + 2)) / 5
```

Comment:
Where are these particular numbers coming from? Are they empirical? Empirical is fine, but it's worth documenting the notes of why we choose these and on what datasets did we assume these numbers gave us something useful.